.github/workflows/gen_pyver_matrix.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import re
+import json
+
+def version_map(py_ver):
+    ci_ver = re.sub("\.", "", py_ver)
+    return f"cp{ci_ver}-*"
+
```

Comment:
Could you add a comment here describing an example output from this script?

Reply:
Yep, good idea!